travis/shared_configs/python-tester-conda.yml

```diff
@@ -33,17 +33,18 @@ jobs:
         - echo "Conda Requirements':' ${CONDA_REQUIREMENTS:=dev-requirements.txt}"
         - echo "Conda packages installed for CI':' ${CONDA_CI_PACKAGES:=pytest-cov}"
         # Manage conda environment
-        - micromamba create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS} --file ${CONDA_REQUIREMENTS}
+        - micromamba create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS} ${CONDA_CI_PACKAGES} --file ${CONDA_REQUIREMENTS}
```

Comment:
`pytest-cov` was missing for `python-tester-conda`-based builds prior to this change